### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by DuckerMan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <a href="#installation">Installation</a> •
   <a href="#customization">Customization</a> •
   <a href="#project-goals-and-alternatives">Project goals, alternatives</a> •
-  Translation [<a href="https://github.com/chinanf-boy/bat-zh">中文</a>][<a href="doc/README-ja.md">日本語</a>][<a href="doc/README-ko.md">한국어</a>]
+  Translation [<a href="https://github.com/chinanf-boy/bat-zh">中文</a>][<a href="doc/README-ja.md">日本語</a>][<a href="doc/README-ko.md">한국어</a>][<a href="doc/README-ko.md">Русский</a>]
```
Comment (sharkdp) [inline README.md:16]: shouldn't this link to README-ru.md?

**Comment by DuckerMan** [comment]
Hi there! Nope, this is a draft

File: README.md (hunk 1 above)
Comment (DuckerMan) [reply README.md:16]: It should :)

**Comment by DuckerMan** [comment]
So.. translation is over!

**Comment by sharkdp** [comment]
Thank you!

**Comment by sharkdp** [comment]
Even the screenshots are translated!! 

Awesome work :trophy:

**Pull request merged**